src/config/SourcegraphReposConfigType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+final class SourcegraphRepoConfigType extends PhabricatorJSONConfigType
+{
+
+    const TYPEKEY = 'sourcegraph.repo';
+
+    public function validateStoredValue(
+        PhabricatorConfigOption $option,
+        $value) {
+        foreach ($value as $index => $spec) {
+            if (!is_array($spec)) {
+                throw $this->newException(
+                    pht(
+                        'Sourcegraph repository configuration is not valid: each entry in ' .
+                        'the list must be a dictionary describing a repository, but ' .
```

Comment:
In the SourcegraphConfigOptions.php you use the term "object" but here you use "dictionary". Which is correct?